packages/driver/cypress/integration/commands/screenshot_spec.js

```diff
@@ -761,9 +761,9 @@ describe('src/cy/commands/screenshot', () => {
         cy.get('.short-element').within(() => {
           cy.screenshot({ capture: 'runner' })
         }).then(() => {
-          // the runner was captured
-          expect(Cypress.action.withArgs('cy:before:screenshot').args[0][1].appOnly).to.be.true
-          expect(Cypress.automation.withArgs('take:screenshot').args[0][1].capture).to.equal('viewport')
+          // the runner was captured ("appOnly === true" means to hide the runner UI)
+          expect(Cypress.action.withArgs('cy:before:screenshot').args[0][1].appOnly).to.be.false
+          expect(Cypress.automation.withArgs('take:screenshot').args[0][1].capture).to.equal('runner')
```

Comment:
According to the definition of `appOnly` below,

https://github.com/cypress-io/cypress/blob/b0fe8bf0dd90d302e51b8d75782edb3b19d5af3e/packages/driver/src/cy/commands/screenshot.ts#L269-L272

it means to hide the runner UI. In other words, it should be `false` to capture the runner UI.  

This weird test failure happened because `appOnly` exists only for testing. 

And the original code was trying to check the options given with `options` after `userOptions` alias is created. 

https://github.com/cypress-io/cypress/blob/b0fe8bf0dd90d302e51b8d75782edb3b19d5af3e/packages/driver/src/cy/commands/screenshot.ts#L444-L455

The code above, when `name` is `{ capture: 'runner' }` and `options` is `undefined`. `options.capture !== 'runner'` is always `true`.